packages/notifi-react-card/lib/context/NotifiSubscriptionContext.tsx

```diff
@@ -55,6 +55,8 @@ export type NotifiSubscriptionData = Readonly<{
   setConversationId: (conversationId: string) => void;
   userId: string;
   setUserId: (userId: string) => void;
+  setSupportAvatarUrl: (supportAvatarUrl: string) => void;
+  supportAvatarUrl: string;
```

Comment:
hmmmmmmm instead of pulling it from the `conversation.participant` I think both of these fields is better to pull from `conversationMessage.conversationParticipant`

For now, I'm okay with it as-is. Sorry I missed this in previous reviews.